extension/chrome/settings/index.ts

```diff
@@ -370,7 +370,8 @@ View.run(
           // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
           const acctRes = await this.acctServer!.accountGetAndUpdateLocalStore();
           $('#status-row #status_flowcrypt').text(`fc:ok`);
-          if (acctRes?.account?.alias) {
+          // if (acctRes?.account?.alias) { // Todo: get alias
+          if (acctRes) {
             statusContainer.find('.status-indicator-text').css('display', 'none');
             statusContainer.find('.status-indicator').addClass('active');
           } else {
```

Comment:
@tomholub Now accountGetAndUpdateLocalStore only returns clientConfigruation and defaultWebPortalMessageExpire.
How this should be handled?
I think this status-indicator is for encrypted contact page?

Reply:
Let's remove the status indicator

Reply:
ok